src/OrbitGl/EventTrack.cpp

```diff
@@ -128,30 +127,24 @@ void EventTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingM
     constexpr const float kPickingBoxOffset = (kPickingBoxWidth - 1.0f) / 2.0f;
 
     auto action_on_callstack_events = [=](const orbit_client_protos::CallstackEvent& event) {
-      uint64_t time = event.time();
-      if (time > min_tick && time < max_tick) {
-        Vec2 pos(time_graph_->GetWorldFromTick(time) - kPickingBoxOffset,
-                 pos_[1] - track_height + 1);
-        Vec2 size(kPickingBoxWidth, track_height);
-        auto user_data = std::make_unique<PickingUserData>(
-            nullptr, [&](PickingId id) -> std::string { return GetSampleTooltip(id); });
-        user_data->custom_data_ = &event;
-        batcher->AddShadedBox(pos, size, z, kGreenSelection, std::move(user_data));
-      }
+      const uint64_t time = event.time();
+      Vec2 pos(time_graph_->GetWorldFromTick(time) - kPickingBoxOffset, pos_[1] - track_height + 1);
+      Vec2 size(kPickingBoxWidth, track_height);
+      auto user_data = std::make_unique<PickingUserData>(
+          nullptr, [&](PickingId id) -> std::string { return GetSampleTooltip(id); });
```

Comment:
Nit: `[&]` -> `[this]` At least to me that helps to understand that `GetSampleTooltip` is a member function